Integrations/integration-IsItPhishing.yml

```diff
@@ -0,0 +1,166 @@
+description: Collaborative web service that provides validation on whether a URL is
+  a phishing page or not by analyzing the content of the webpage
+configuration:
+- display: Customer's name
```

Comment:
are those fields that long they need the long text type?
and usually, URL is first